bin/main.rb

```diff
@@ -0,0 +1 @@
+puts 'Hello world!'
```

Comment:
- [ ] `main.rb` is [executable file](https://commandercoriander.net/blog/2013/02/16/making-a-ruby-script-executable/):
* `#!/usr/bin/env ruby` is the top line
* puts "Hello world" in the following line ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/OOP/milestone1/execute_bin.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59565018/))

Please add the top line

Reply:
there is lib folder in our folder structure and it is executable using the `bin/main.rb` command on bash. please kindly check again.